MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,916 @@
+
+DataGridImageColumn is derived from DataGridColumn. Hence, it inherits all the properties of DataGridColumn. It displays image as cell content of a column. To create DataGridImageColumn, the property corresponding to the column in the underlying collection must be `ImageSource` type.
+
+In DataGridImageColumn, it is possible to load images in any of the following four ways:
+
+ * **FromFile**: Required to specify the path of the file.
+ * **FromResource**: Required to set image as embedded resource.
+ * **FromStream**: Required to load image from byte[] array.
+ * **FromURI**: Required to set image from a web service or website.
+
+To load image (embedded resource) in DataGridImageColumn, follow the code example:
+ 
+{% highlight xaml %}
+<ContentPage.BindingContext>
+    <local:ViewModel />
+</ContentPage.BindingContext>
+
+<sfGrid:SfDataGrid x:Name="dataGrid"
+                   AutoGenerateColumnsMode="None"
+                   ItemsSource="{Binding OrdersInfo}">
+    <sfGrid:SfDataGrid.Columns>
+        <sfGrid:DataGridImageColumn MappingName="DealerImage" />
+    </sfGrid:SfDataGrid.Columns>
+</sfGrid:SfDataGrid>
+{% endhighlight %}
+
+{% highlight c# %}
+// Model class
+public class Model
+{
+    private ImageSource _dealer;
+
+    public ImageSource DealerImage
+    {
+        get { return _dealer; }
+        set
+        {
+            this._dealer = value;
+        }
+    }
+}
+
+// ViewModel class
+public class ViewModel
+{
+    public ViewModel()
+    {
+        GetOrderDetails(50);
+    }
+
+    #region ItemsSource
+
+    private ObservableCollection<OrderInfo> ordersInfo;
+
+    public ObservableCollection<OrderInfo> OrdersInfo
+    {
+        get { return ordersInfo; }
+        set { this.ordersInfo = value; }
+    }
+
+    #endregion
+
+    #region ItemSource Generator
+
+    public void GetOrderDetails(int count)
+    {
+        var orderDetails = new ObservableCollection<OrderInfo>();
+        for (int i = 1; i <= count; i++)
+        {
+            var order = new OrderInfo()
+            {
+                DealerImage = ImageSource.FromResource("DataGridDemo.Buchanan.png") // Need to give the image path properly. Here, DataGridDemo denotes the project name and Buchanan denotes the image name.
+            };
+            orderDetails.Add(order);
+        }
+        ordersInfo = orderDetails;
+    }
+
+    #endregion
+}
+{% endhighlight %}
+
+### Aspect
+
+SfDataGrid allows you to set the `Aspect` to size the loaded images within the bounds of the grid cell (whether to stretch, crop or letterbox) using the `DataGridImageColumn.Aspect` property. The supported aspects are described below, the default value is AspectFit.
+
+AspectFill: Clips the image so that it fills the display area while preserving the aspect (no distortion).
+
+AspectFit:  Letterboxes the image (if required) so that the entire image fits into the display area, with blank space added to the top/bottom or sides depending on whether the image is wide or tall.
```

Comment:
AspectFit:  Letterboxes the image (if required) so that the entire image fits into the display area, with a blank space added to the top/bottom or sides depending on whether the image is wide or tall.